lib/ansible/modules/storage/netapp/na_ontap_dns.py

```diff
@@ -40,6 +40,13 @@
     description:
     - List of IPv4 addresses of name servers such as '123.123.123.123'.
 
+  skip_validation:
+    type: bool
+    description:
+    - By default, all nameservers are checked to validate they are available to resolve.
+    - If you DNS servers are not yet installed or momentarily not available, you can set this option to 'true'
```

Comment:
```suggestion
    - If your DNS servers are not yet installed or momentarily not available, you can set this option to I(true).
```

Reply:
Apparently, I don't have permissions to push to your branch. I didn't want to hold up merge. Feel free to raise a PR to fix this if you'd like, though don't lose sleep over it